### PR TITLE
fix(git): resync 21.10.x to dev-21.10.x

### PR DESCRIPTION
**Pull request by kduret**
## Description

resync 21.10.x to dev-21.10.x

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [x] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x (master)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+/.git*          @centreon/centreon-devops
+/.project       @centreon/centreon-devops
+/Jenkinsfile    @centreon/centreon-devops
+/selinux/       @centreon/centreon-devops
+/project/       @centreon/centreon-devops
+*.sh            @centreon/centreon-devops
 
-*.po            @ghost
+/.snyk				@centreon/centreon-security
+/sonar-project.properties	@centreon/centreon-security
```
Comment (Thebarda) [inline .github/CODEOWNERS:10]: It seems the team @centreon/centreon-security does ot exist

File: www/front_src/src/Resources/Actions/Resource/Downtime/index.tsx [hunk 2]
```diff
@@ -67,6 +67,8 @@ const DowntimeForm = ({
   });
   const { alias, downtime } = useUserContext();
 
+  const { alias, downtime } = useUserContext();
+
```
Comment (Thebarda) [inline www/front_src/src/Resources/Actions/Resource/Downtime/index.tsx]:
```suggestion
```

Comment (kduret) [inline www/front_src/src/Resources/Actions/Resource/Downtime/index.tsx]:
```suggestion
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (kduret) [reply .github/CODEOWNERS:10]: @bdauria team must be created ? or those lines must be deleted ?

Comment (sc979) [reply .github/CODEOWNERS:10]: the codeowner will be modified next week to reflect latest changes.
The security team will be created in october

Comment (bdauria) [reply .github/CODEOWNERS:10]: This will be handled later on

**Review by jdelpierre — APPROVED** [review]

File: src/Centreon/Application/Controller/MonitoringResourceController.php [hunk 3]
```diff
@@ -100,6 +100,11 @@ class MonitoringResourceController extends AbstractController
     private const HOSTGROUP_CONFIGURATION_URI = '/main.php?p=60102&o=c&hg_id={resource_group_id}';
     private const SERVICEGROUP_CONFIGURATION_URI = '/main.php?p=60203&o=c&sg_id={resource_group_id}';
 
+    private const HOSTGROUP_CONFIGURATION_URI = '/main.php?p=60102&o=c&hg_id={resource_group_id}';
+    private const SERVICEGROUP_CONFIGURATION_URI = '/main.php?p=60203&o=c&sg_id={resource_group_id}';
+    private const SERVICE_REDIRECT_URL_ENDPOINT = 'centreon_application_monitoring_resource_redirect_url_service';
+    private const HOST_REDIRECT_URL_ENDPOINT = 'centreon_application_monitoring_resource_redirect_url_host';
+
```
Comment (callapa) [inline src/Centreon/Application/Controller/MonitoringResourceController.php]: Already defined
```suggestion
```

**Review by callapa — CHANGES_REQUESTED** [review]

**Review by callapa — APPROVED** [review]

**Pull request merged**